### PR TITLE
i18n(fr): Add `integrations-guide/deno` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
i18n(fr): Add `integrations-guide/deno` in French language

#### Related issues & labels (optional)

- Linked to https://github.com/withastro/docs/issues/6052
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/integrations-guide/deno.mdx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+Vous pouvez maintenant utiliser cette commande pour prévisualiser votre site Astro de production localement avec Deno.
+
+ ```bash
+ npm run preview
+    ```
+
+## Comment déployer
+
+You can deploy to Deno Deploy through GitHub Actions or using Deno Deploy’s CLI (command line interface).
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/integrations-guide/deno.mdx:75]:
```suggestion

```

File: src/content/docs/fr/guides/integrations-guide/deno.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+       - name: Téléchargement vers Deno Deploy
+         uses: denoland/deployctl@v1
+         with:
+           project: my-deno-project # TODO: remplacer par le nom du projet Deno Deploy
+           entrypoint: server/entry.mjs
+           root: dist
+    ```
+
+1. Après avoir validé ce fichier YAML, et l'avoir poussé sur GitHub sur votre branche de déploiement configurée, le déploiement devrait commencer automatiquement !
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/integrations-guide/deno.mdx:123]:
```suggestion
1. Après avoir validé ce fichier YAML et l'avoir poussé sur votre branche de déploiement configurée, le déploiement devrait commencer automatiquement !
```

File: src/content/docs/fr/guides/integrations-guide/deno.mdx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+1. Exécutez `deployctl` pour déployer !
+
+Dans la commande ci-dessous, remplacez `<ACCESS-TOKEN>` par votre [Personal Access Token](https://dash.deno.com/account#access-tokens) et `<MY-DENO-PROJECT>` par le nom de votre projet Deno Deploy.
+ ```bash
+ DENO_DEPLOY_TOKEN=<ACCESS-TOKEN> deployctl deploy --project=<MY-DENO-PROJECT> --no-static --include=./dist ./dist/server/entry.mjs
+    ```
+
+Vous pouvez suivre tous vos déploiements sur [Deno Deploy](https://dash.deno.com).
+
+1) (Optionnel) Pour simplifier la construction et le déploiement en une seule commande, ajoutez un script `deploy-deno` dans `package.json`.
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/integrations-guide/deno.mdx:150]:
```suggestion
1. (Optionnel) Pour simplifier la construction et le déploiement en une seule commande, ajoutez un script `deploy-deno` dans `package.json`.
```

**Review by McFlyPartages — COMMENTED** [review]
Several suggestions but good works.

File: src/content/docs/fr/guides/integrations-guide/deno.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/deno.mdx]: Oh merci, j'ai oublié de supprimer la version anglaise..

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/fr/guides/integrations-guide/deno.mdx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+           root: dist
+    ```
+
+1. Après avoir validé ce fichier YAML et l'avoir poussé sur votre branche de déploiement configurée, le déploiement devrait commencer automatiquement !
+
+Vous pouvez suivre la progression en utilisant l'onglet "Actions" sur votre page de dépôt GitHub, ou sur [Deno Deploy](https://dash.deno.com).
+
+### Déploiement CLI
+
+1. Installez le [Deno Deploy CLI] (https://docs.deno.com/deploy/manual/deployctl).
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/integrations-guide/deno.mdx:128]:
```suggestion
1. Installez le [Deno Deploy CLI](https://docs.deno.com/deploy/manual/deployctl).
```

**Comment by McFlyPartages** [comment]
LGTM 😛

**Review by Yan-Thomas — CHANGES_REQUESTED** [review]
The translated page here is `/guides/deploy/deno`, but is in the place of the `integrations-guide/deno` place. The translation seems fine, but we need to move it to the correct file path.

**Review by Yan-Thomas — APPROVED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**